libscript/src/script-execute.cpp

```diff
@@ -202,10 +209,32 @@ class MCScriptForeignInvocation
             break;
         
         case kMCScriptForeignHandlerLanguageC:
```

Comment:
We should probably throw an error if a variadic foreign handler binds to java since it's not yet supported